examples/feature_demo/line_basic.py

```diff
@@ -2,7 +2,6 @@
 Line Drawing
 ============
 
-
 Some basic line drawing.
```

Comment:
this docstring carries a different meaning now :)

Reply:
Updated.